iaso/api/export_requests.py

```diff
@@ -65,9 +66,13 @@ def create(self, validated_data: typing.MutableMapping):
             raise serializers.ValidationError({"code": type(e).__name__, "message": str(e)})
 
     def update(self, export_request, validated_data):
-        DataValueExporter().export_instances(export_request)
-        # this has a highly probable chance to timeout but the export will continue to be processed
-        # still return the export request
+        try:
+            DataValueExporter().export_instances(export_request)
+        except InstanceExportError as e:
```

Comment:
I hope that InstqnceExportError are wrapping the dhis2.exceptions.RequestException